mmdet3d/models/necks/view_transformer.py

```diff
@@ -0,0 +1,347 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn as nn
+from mmcv.runner import BaseModule
+
+from ..builder import NECKS
+
+
+class QuickCumsum(torch.autograd.Function):
+    """Sum up the features of all points within the same voxel through
+    cumulative sum operator.
+
+    All inputs should be sorted by the rank of voxels.
+
+    The function implementation process is as follows:
+
+        - step 1: Cumulatively sum the point-wise feature alone the point
+            queue.
+        - step 2: Remove the duplicated points with the same voxel rank and
+            only retain the last one in the point queue.
+        - step 3: Subtract each point feature with the previous one to obtain
+            the cumulative sum of the points in the same voxel.
+
+    Args:
```

Comment:
Please move the arguments below forward. The arguments docstring of a class typically corresponds to the args of the init function.